src/compute/src/communication.rs

```diff
@@ -82,11 +82,15 @@ pub async fn initialize_networking(
     }
 
     match initialize_networking_from_sockets(sockets, *process, *workers, Box::new(|_| None)) {
-        Ok((stuff, guard)) => Ok((
-            stuff.into_iter().map(GenericBuilder::ZeroCopy).collect(),
-            Box::new(guard),
-        )),
+        Ok((stuff, guard)) => {
+            info!(process = process, "successfully initialized network");
+            Ok((
+                stuff.into_iter().map(GenericBuilder::ZeroCopy).collect(),
+                Box::new(guard),
+            ))
+        }
         Err(err) => {
+            error!(process = process, "failed to initialize network: {err}");
```

Comment:
`error!` will report to Sentry. Is that the right thing to use here? We're trying to limit `error!`s to stuff that will one day page the on call engineer.

Reply:
no, it can be a warning